src/card/basic/card.directive.js

```diff
@@ -3,26 +3,140 @@
  * @name patternfly.card.directive:pfCard
  * @restrict A
  * @element ANY
- * @param {headTitle=} Title for the card - required
- * @param {subTitle=} Subtitle for the card - optional
- * @param {showTopBorder=} Show Top Border, true shows top border, false (default) hides top border - optional
- *
+ * @param {string} headTitle Title for the card
+ * @param {string=} subTitle Sub-Title for the card
+ * @param {boolean=} showTopBorder Show/Hide the blue top border. True shows top border, false (default) hides top border
+ * @param {boolean=} showTitlesSeparator Show/Hide the grey line between the title and sub-title.
+ * True (default) shows the line, false hides the line
+ * @param {object=} footer footer configuration properties:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.iconClass  - (optional) the icon to show on the bottom left of the footer panel
+ * <li>.text       - (optional) the text to show on the bottom left of the footer panel, to the right of the icon
+ * <li>.href       - (optional) the href link to navigate to when the footer href is clicked
+ * <li>.callBackFn - (optional) user defined function to call when the footer href is clicked
+ * </ul>
+ * *Note: If a href link and a callBackFn are specified, the href link will be called
+ * @param {object=} filter filter configuration properties:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.filters    - drop down items for the filter.
+ *<pre class=''>
+ *  Ex:  'filters' : [{label:'Last 30 Days', value:'30'},
+ *                    {label:'Last 15 Days', value:'15'},
+ *                    {label:'Today', value:'today'}]</pre>
+ * <li>.defaultFilter - integer, 0 based index into the filters array
+ * <li>.callBackFn - user defined function to call when a filter is selected
+ * </ul>
  * @description
  * Directive for easily displaying a card with html content
  *
  * @example
- <example module="patternfly.card">
+ <example module="test">
 
  <file name="index.html">
-    <div pf-card head-title="My Card Title" sub-title="My card subtitle">
-      <button>Click Me</button>
-    </div>
+   <div ng-controller="ChartCtrl">
+     <div pf-card head-title="Card Title" sub-title="Card Subtitle" filter="filterConfigHeader" style="width: 50%">
+       [Card Contents] <button>Click Me</button> Timeframe filter in header
+     </div>
+
+     <div pf-card head-title="Card Title" sub-title="Card Subtitle" show-top-border="true"
+          footer="footerConfig" filter="filterConfig" style="width: 50%">
```

Comment:
Same reason as above, it's just for this ngDoc example.  Each example may have different layouts, widths, etc..